app/schema.py

```diff
@@ -131,3 +131,32 @@ class Feedback(BaseModel):
 class Resource(BaseModel):
     name: constr(max_length=255)
     item_id: constr(max_length=255)
+
+
+class Comments(BaseModel):
+    comment_id: constr(max_length=255)
+    comment_text: Optional[constr(max_length=2000)]
+    created_at: datetime
+    note_id: constr(max_length=255)
+    updated_at: datetime
+
+
+class Notes(BaseModel):
```

Comment:
Add a docstring including that this is comparable to our "Memos," 
and that a better name would be MeetingNotes because it's meant for the notes specifically related to a meeting. Also include any relations from the Postgres db as seen on the milanote diagram. ("Notes" from postgres is separate from our "Feedback.")